lib/02_calculator.rb

```diff
@@ -0,0 +1,13 @@
+def add(f,s)
+  add = f + s
+end
+
+def subtract(f,s)
+  sub = f - s
```

Comment:
`sub =` is not needed here but you can also see that `sub` is highlighted in blue which means it's a ruby method. That means you shouldn't be using that when naming a variable anyway, because it's one of those "special" words. If you were to pick a variable name, I would go with `subtract` or anything that is not already the name of an existing ruby method.